lib/internal/process/promises.js

```diff
@@ -120,7 +146,13 @@ function emitPromiseRejectionWarnings() {
   return maybeScheduledTicks || pendingUnhandledRejections.length !== 0;
 }
 
+function listenForRejections() {
+  setPromiseRejectCallback(promiseRejectHandler);
```

Comment:
Shouldn't it be fine to just execute this right away instead of exporting this and executing while the other setup is executed (which should be at the same time)?

Reply:
I think the cleaner solution would be to just merge promises.js with next_tick.js - otherwise it would be odd to have the knowledge of how to set up the rejection callback in next_tick.js: what's the point of having the files split if the logic is not?

Reply:
I would be fine with merging the two files.